server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -1492,6 +1634,15 @@ public void testRetrieveUnusedSegmentsWithMaxUsedStatusLastUpdatedTime2() throws
     );
     Assert.assertEquals(oddYearSegments.size(), actualUnusedSegments1.size());
 
+    final ImmutableList<DataSegmentPlus> actualUnusedSegmentDtos1 = retrieveUnusedSegmentsPlus(
```

Comment:
Stale references in this file here and below:
```suggestion
    final ImmutableList<DataSegmentPlus> actualUnusedSegmentsPlus1 = retrieveUnusedSegmentsPlus(
```

Reply:
fixed